New enrich your assets (#433)
akeneo/pim-helpcenter: content/md/products-super-power/enrich-your-assets.md

---
id: enrich-your-assets
themes: products-super-power, manage-your-assets
title: "Enrich your **assets**"
popular: false
ee-only: right
related:
---

# Add an asset
Once you have created an asset family, you can add **assets**.

> In the asset manager, you have 2 ways to add assets: unitary (one by one) or via the [mass upload](mass_upload.html) feature.

Reviewer: Je mettrai une liste avec des pu ces pour qu'on voit bien les 2 modes

Let’s say that you just have created the asset family `Packshots`, you have defined its asset attributes and now it is time to add new assets.

To add a new asset to the asset family `Packshots`:
1. Go to the Asset Manager
1. Select your `Packshots` family (the assets will automatically appear in the grid)
1. Then click on `Create` and select `Create an asset`.
1. Choose a label for the asset (the PIM will automatically take its label to generate your asset's code, but you can change it if needed).

> ![Create an asset](../img/Asset_creation.png)

Reviewer: Les images ne s'affichent pas, je sais pas sir le pb est de mon côté ou du tien :(

For example, if you search for the word *« famous »*, all the assets containing the word *« famous »* in their code, label or text attribute types will be displayed.  

![Search for assets with a keyword](../img/Assets_Search_1_word.png)

And we can go further.
If you search for *"famous"* and *"bags"*, then only the record containing both *"famous"* and *"bags"* words will be displayed.

![Search for assets with several keywords](../img/Assets_Search_2_words.png)

## Filter on options

There are also filters on the left, where you retrieve the options of the single/multiple options attributes. It can be very powerful if you use these options to manage tags on your assets for example.

![Asset library](../img/Asset_filter_options.png)

The operator is an `in list` operator. If you are looking for the packshot with the room "kitchen" OR "leaving room", the packshots with the room "kitchen" OR the room "leaving roomd" will be displayed.

You can combine filters. For instance, you can search on all packshots with the material "wood" AND the color "brown".

An option can be removed from a filter using the cross.

To empty a filter, you can click on the eraser.

> # Edit an asset

Reviewer: Enrich better than edit? Peut-être prendre des exemples d'enrichissement d'asset.

---
id: enrich-your-assets
themes: products-super-power, manage-your-assets
title: "Enrich your **assets**"
popular: false
ee-only: right
related:
---

# Add an asset
Once you have created an asset family, you can add **assets**.
In the asset manager, you have 2 ways to add assets: unitary (one by one) or via the [mass upload](mass_upload.html) feature.
Let’s say that you just have created the asset family `Packshots`, you have defined its asset attributes and now it is time to add new assets.

To add a new asset to the asset family `Packshots`:
1. Go to the Asset Manager
1. Select your `Packshots` family (the assets will automatically appear in the grid)
1. Then click on `Create` and select `Create an asset`.
1. Choose a label for the asset (the PIM will automatically take its label to generate your asset's code, but you can change it if needed).

![Create an asset](../img/Asset_creation.png)

::: info
Note that we have defined a limitation of 1 million assets per asset family to guarantee that the PIM is functional and runs smoothly.
:::

> # The asset library

Reviewer: Sur ce, l'asset library se trouve dans les 2 articles, manage et enrich, car avant il y avait 2 écrans pour les reference entities. Dans le manage, il faudrait peut-être plus rester sur la structure.

Once the assets are added to the family, they appear in a mosaic view, that we call the **asset library**.

> ![Asset library](../img/Asset_library.png)

Reviewer: Préciser ce qui est afficher au niveau du thumbnail de l'asset (image, complétude...)

---
id: work-on-your-assets
themes: products-super-power, work-on-your-assets
title: "Work on your **assets**"
popular: false
ee-only: right
related:
---

# Enrich an asset
Once you have created your asset, you can enrich it in order to add some information.
Nothing simpler: just go on an asset and add/change the values you want.  

Let's take an example.

You are in your `videos` asset family, in the `alpha` asset. You want to add the **Youtube link** of the video, the **capture image** of this video, a **short description**, a **duration** in seconds. To do that, the only prerequisite is to create those 4 attributes in your `video` asset family. Then, you can add all the content you want in these attributes ;)

> 

Reviewer: Ca peut peut-être  valoir le coup de reparler du rich text editor comme pour les records des ref entities avec la description en exemple

# Follow the assets completeness

Like for products, we calculate [completeness](manage-assets.html) on assets.

In the assets, several indicators show you the status of your completeness and which attributes are missing.
- **Yellow bullet points** in front of each missing required attribute, showing that you have to complete this attribute to improve the completeness.
- The **completeness percentage** in the header of the asset informs you of the completeness level.   

You also have the completeness indicator directly on the asset thumbnail in the library.

The completeness is **automatically calculated** as soon as we enrich a required attribute. It means that the bullet point will disappear as soon as the required attribute is filled, and same for the completeness percentage evolution.

# Delete assets

## Delete an asset
You can delete an asset or all the assets of an asset family.
 To delete an asset, click on the asset, then on the `...` button at the top right corner of the header, and `delete`.

## Delete all the assets
To delete all the assets of a family, click on the `...` button at the top right corner of the header's asset library, then `delete all assets`.

::: tips

> To delete all the assets of an asset family, you **need to check that no asset of this family is used in a product**. Otherwise you will not be able to delete it.

Reviewer: ```suggestion
To delete all the assets of an asset family, you **need to check that no asset of this family is linked to a product or a product model**. Otherwise you will not be able to delete it.
```
